CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- Every sampling method returns created_at timestamps as the last argument.
```

Comment:
In the docstrings it says timestamps are only returned for temporal graphs but here it looks like they are always returned.

Reply:
I need to update docstrings, thank!

Reply:
Can you add a parameter to enable returning timestamps, I believe this is standard practice instead of returning many people info they do not need: https://numpy.org/doc/stable/reference/generated/numpy.unique.html

Reply:
Make sense, Let me add it.

Reply:
Fixed.